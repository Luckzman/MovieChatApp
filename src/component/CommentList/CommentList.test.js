import React from 'react';
import { shallow } from 'enzyme';
import { CommentList } from './';

describe('CommentList', () => {
  it('should render correctly', () => {
    const props = {
      comments: {data: ['cool', 'calm'] }
    }
    const wrapper = shallow(< CommentList {...props} />);

    expect(wrapper).toMatchSnapshot();
  });
});
