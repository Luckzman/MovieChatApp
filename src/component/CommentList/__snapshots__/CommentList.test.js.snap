// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentList should render correctly 1`] = `ShallowWrapper {}`;
